src/components/ClassOverview.vue

```diff
@@ -19,12 +19,18 @@
 					<li
 						v-for="property in visibleProperties"
 						:key="scopedName(property)"
-						class="cursor-pointer mb-14"
+						class="
+							cursor-pointer
+							mb-14
+							border-l-4 border-transparent
+							rounded-sm
+							hover:border-l-4 hover:border-discord-blurple-500
+						"
 						@click="scrollTo(scopedName(property))"
 					>
 						<router-link
 							:to="{ name: 'docs-source-tag-class-class', query: { scrollTo: scopedName(property) } }"
-							class="mr-2"
+							class="pl-1 mr-2"
```

Comment:
One more thing I noticed in the preview, can we make this `mx-2` instead? (And remove the `pl-1`)